mars/web/api.py

```diff
@@ -129,10 +130,14 @@ def delete(self, session_id, graph_key):
 class GraphDataHandler(ApiRequestHandler):
     @gen.coroutine
     def get(self, session_id, graph_key, tensor_key):
-        data_type = self.request.arguments.get('type')
-        if data_type == 'nsplits':
-            nsplits = self.web_api.get_tensor_nsplits(session_id, graph_key, tensor_key)
-            self.write(json.dumps(nsplits))
+        type_argument = self.request.arguments.get('type')
+        if type_argument:
+            data_type = to_str(type_argument[0])
+            if data_type == 'nsplits':
```

Comment:
No tests for this branch?

Reply:
They are covered by case in `test_cluster.testBoolIndexingExecute`.